File: 0884-uncommon-words-from-two-sentences/0884-uncommon-words-from-two-sentences.java
class Solution {
    HashMap<String, Integer> m= new HashMap<>();
    void hashConvertor(String[] s1){
        for( int i=0;i<s1.length;i++){
            if (m.containsKey(s1[i])){
                m.put(s1[i],m.get(s1[i])+1);
            }
            else{
                m.put(s1[i],1);
            }
        }
    }
    public String[] uncommonFromSentences(String s1, String s2) {
        String [] s=s1.split(" ",0);
        String [] ss=s2.split(" ",0);
        
        hashConvertor(s);
        hashConvertor(ss);
        
        ArrayList<String>v=new ArrayList<String>();
        for(Map.Entry<String,Integer> i:m.entrySet()){
            if(i.getValue()==1){
                v.add(i.getKey());
            }
        }
        String[] aa =new String[v.size()];
        int j=0;
        for(String i:v){
            aa[j]=i;
            j++;
        }
        return aa;

        
        
    }
}

File: 0884-uncommon-words-from-two-sentences/0884-uncommon-words-from-two-sentences.py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        d={}
        l=s1.split(" ")
        l1=s2.split(" ")
        for i in l:
            if i in d:
                d[i]+=1
            else:
                d[i]=1
        for i in l1:
            if i in d:
                d[i]+=1
            else:
                d[i]=1
        ans=[]
        for i in d.keys():
            if d[i]==1:
                ans.append(i)
        return ans
        